src/main/java/mikera/matrixx/impl/SparseRowMatrix.java

```diff
@@ -209,6 +237,22 @@ public AVector getRow(int i) {
 		if (v == null) return emptyRow;
 		return v;
 	}
+
+    @Override
+	public List<AVector> getSlices() {
+        // TODO: Is it acceptable for views to be mixed with
```

Comment:
I'm not sure what my concern was here. @mikera, do you see any problem with this?